### PR TITLE
updated-operators

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/rules/cse-rules-syntax.md [hunk 1]
```diff
@@ -1162,6 +1162,19 @@ Filters results based on the value of a boolean expression.  
 
 `| where jsonArrayContains(field, “vuln_scanner”)`  
  
+## in
```
Comment (kimsauce) [inline docs/cse/rules/cse-rules-syntax.md:1165]: It looks like this page is in alphabetical order, so you'll need to move "in" to the right spot

**Review by kimsauce — CHANGES_REQUESTED** [review]
place "in" operator in ABC order

File: docs/cse/rules/cse-rules-syntax.md (hunk 1 above)
Comment (JainM6) [reply docs/cse/rules/cse-rules-syntax.md]: @kimsauce: Thank you so much for pointing out this. I have fixed it.

**Review by kimsauce — APPROVED** [review]

File: docs/cse/rules/cse-rules-syntax.md (hunk 1 above)
Comment (kimsauce) [reply docs/cse/rules/cse-rules-syntax.md]: thank you!

**Pull request merged**